astropy/table/operations.py

```diff
@@ -236,7 +236,7 @@ def vstack(tables, join_type='outer', metadata_conflicts='warn'):
 
     Parameters
     ----------
-    tables : Table or list of Table objects
+    tables : Table or list of Table and/or Row objects
```

Comment:
You've made a distinction between ``hstack`` and ``vstack`` as follows:
``Column`` is allowed for ``hstack`` and ``Row`` is allowed for ``vstack`` which makes sense. But the issue is that both call the same function ``_get_list_of_tables`` to validate the input and hence both will accept Rows as well as Columns. If the distinction is necessary, you will have to validate the input accordingly; probably by using a keyword argument in ``_get_list_of_tables`` to make it backwards compatible or separately checking for Column for ``hstack`` (just a suggestion).

Reply:
got the point.. will work on it.